### PR TITLE
refactored sync functions and utility file --> importer file to make …

### PR DESCRIPTION
**Pull request by petmetic**
…it easier to implement API call for 30days ahead

### CODE REVIEW

File: src/web/wordpress_api_calls.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    wp_json = session.get(url).text
+    wp_massage_customer = json.loads(wp_json)
+
+    return wp_massage_customer
+
+
+def get_therapist_service_data_from_wp(nonce, session):
+    url = settings.WP_URL_ENTITIES.format(nonce=nonce)
+    wp_json = session.get(url).text
+    wp_therapist_service = json.loads(wp_json)
```
Comment (jurecuhalev) [inline src/web/wordpress_api_calls.py:54]: you can use wp_json.data()

**Review by jurecuhalev — APPROVED** [review]

File: src/web/wordpress_api_calls.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        "%Y-%m-%d"
+    )
+
+    url = settings.WP_URL_APPOINTMENTS.format(
+        nonce=nonce,
+        date_sync_before=date_sync_before,
+        date_sync_week=date_sync_week,
+    )
+    wp_json = session.get(url).text
+    wp_massage_customer = json.loads(wp_json)
```
Comment (jurecuhalev) [inline src/web/wordpress_api_calls.py:46]:
```suggestion
    wp_massage_customer = session.get(url).data()
```

File: src/web/wordpress_api_calls.py (hunk 1 above)
Comment (jurecuhalev) [inline src/web/wordpress_api_calls.py:54]:
```suggestion
    wp_therapist_service = session.get(url).json()
```

File: src/web/wordpress_api_calls.py (hunk 2 above)
Comment (petmetic) [reply src/web/wordpress_api_calls.py:46]: From where do you get data() ?
should it not be: `wp_massage_customer = json.loads(session.get(url).text)` ?

**Pull request merged**